pkg/objstore/swift/swift.go

```diff
@@ -244,8 +245,22 @@ func (c *Container) deleteContainer(name string) error {
 	return containers.Delete(c.client, name).Err
 }
 
-func configFromEnv() SwiftConfig {
-	c := SwiftConfig{
+type testSwiftConfig struct {
```

Comment:
I don't like how we handle this. We can improve this flow we don't need this redundancy.